sdk/ai/azure-ai-client/samples/agents/sample_agents_file_search.py

```diff
@@ -0,0 +1,86 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import os, time, logging
+from azure.ai.client import AzureAIClient
+from azure.ai.client.models._patch import FileSearchTool, ToolSet
+from azure.identity import DefaultAzureCredential
+
+# Set logging level
+logging.basicConfig(level=logging.INFO)
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
+# Customer needs to login to Azure subscription via Azure CLI and set the environment variables
+
```

Comment:
Please use this "standard" header as you see in inference & endpoints samples. Put it below the copyright note.

"""
FILE: sample_get_chat_completions_client.py
DESCRIPTION:
    Given an AzureAIClient, this sample demonstrates how to get an authenticated 
    async ChatCompletionsClient from the azure.ai.inference package.
USAGE:
    python sample_get_chat_completions_client.py
    Before running the sample:
    pip install azure.ai.client azure-identity
    Set this environment variables with your own values:
    AI_CLIENT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
"""

Reply:
Done